### PR TITLE
[usecase] Autoamtion BrowserState and Notifications sample app

### PR DESCRIPTION
**Pull request by JianfengXu**
Impacted TCs num(approved): New 0, Update 2, Delete 0
Unit test Platform: Crosswalk Project for Android 15.44.374
Unit test result summary: Pass 2, Fail 0, Blocked 0

### CODE REVIEW

File: usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Feature: Usecase WebAPI
+ Scenario: Device & Hardware/BrowserState Test
+    When launch "usecase-webapi-xwalk-tests"
+     And I go to "/samples/BrowserState/index.html"
+     And I open wifi
+     And I wait 20 seconds
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature]: Why you wait for so long time?

**Comment by Honry** [comment]
Please update the summary, you updated two tests.

File: usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Feature: Usecase WebAPI
+ Scenario: Device & Hardware/BrowserState Test
+    When launch "usecase-webapi-xwalk-tests"
+     And I go to "/samples/BrowserState/index.html"
+     And I open wifi
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature]: Why not try open/close airplane mode?

File: usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature (hunk 1 above)
Comment (JianfengXu) [inline usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature]: WIFI connect to  internet need some times, wait 20 seconds to ensure that the connection is successful.

File: usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature (hunk 2 above)
Comment (JianfengXu) [inline usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature]: If the WIFI status is closed, open/close airplane mode can not connect to internet.

Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature]: Network connection should be a precondition in usecase-webapi tesing.

File: usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Feature: Usecase WebAPI
+ Scenario: Device & Hardware/BrowserState Test
+    When launch "usecase-webapi-xwalk-tests"
+     And I go to "/samples/BrowserState/index.html"
+     And I open airplane mode
+     And I wait 10 seconds
+     And I execute command "adb shell am force-stop com.android.settings"
+    Then I should see "online status : false"
+    Then I should not see "N/A"
+     And I close airplane mode
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature]: Here we have a risk that if one of above steps fails, the airplane mode is open which would impact other tests.

**Comment by Honry** [comment]
Why is there a fail one while you just update 2 tests?

**Comment by Honry** [comment]
LGTM

**Pull request merged**